README.md

```diff
@@ -203,8 +230,37 @@ To include all the files in a directory, use the `*` wild card (eg. `/var/log/*`
 
 Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
 
-### Monasca-ui
+### Uninstalling Logs agent
+In the following description, 
+[target_dir] is the target directory specified at the time of running log-agent-[version].run
+
+stop the service:
+```
+systemctl stop monasca-log-agent
+```
+
+delete obsolete files: 
```

Comment:
I think `[target_dir]` means e.g. `/opt/monasca-log-agent` and under the directory, following files exist.

- bin
- conf
- logstash-[version]

So change as below.
`rm -r [target_dir]/bin`
`rm -r [target_dir]/conf`
`rm -r [target_dir]/monasca-log-agent`

BTW, isn't it good to remote `[target_dir]`?

Reply:
Done